src/game/Game.vue

```diff
@@ -1,11 +1,17 @@
 <script setup lang="ts">
-import GameRow from './GameRow.vue';
+import GameRow from './GameRow.vue'
 </script>
 
 <template>
-  <GameRow v-for="i in 6" :key="i" :row="i-1"/>
+  <div class="grid">
+    <GameRow v-for="i in 6" :key="i" :row="i - 1" />
+  </div>
 </template>
 
 <style scoped>
-
-</style>
\ No newline at end of file
+.grid {
+  display: grid;
+  grid-template-rows: repeat(6, 1fr);
+  gap: var(--tile-gap);
+}
```

Comment:
I added this to better address the board as a collection of rows. This along with a gap property that so we get block level spacing as well.